packages/core/content-type-builder/admin/src/components/AttributeIcon.tsx

```diff
@@ -86,7 +88,7 @@ export const AttributeIcon = ({ type, customField = null, ...rest }: AttributeIc
   }
 
   return (
-    <IconBox height={`${24 / 16}rem`} width={`${32 / 16}rem`} shrink={0} {...rest} aria-hidden>
+    <IconBox width="3.2rem" shrink={0} {...rest} aria-hidden>
```

Comment:
height is missing

Reply:
not necessarily needed since the heigh & width are in unison now 👍🏼 i.e. the icon doesn't want to be 24px tall because it looks weird